keyboards/h0oni/hotduck/hotduck.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_all( \
+      K00,  K01,  K02,  K03,  K04,  K05,  K06,  K07,  K08,  K09,\
+      K10,  K11,  K12,  K13,  K14,  K15,  K16,  K17,  K18,  K19,\
+      K20,  K21,  K22,  K23,  K24,  K25,  K26,  K27,  K28,  K29,\
+      K30,  K31,  K32,  K33,  K34,  K35,  K36,  K37,  K38,  K39,\
+      K40,  K41,  K42,  K43,  K44,  K45,  K46,  K47,  K48,  K49,\
+      K50,  K51,  K52,  K53,  K54,  K55,  K56,  K57,  K58,  K59,\
+      K60,  K61,  K62,  K63,  K64,  K65,  K66,  K67,  K68,  K69\
+) { \
```

Comment:
The keymap looks like this may have a full qwerty layout.   This doesn't reflect that, unfortunately. 

This block can be (and should be) reformatted to match the key layout of the keyboard.   

You can see boards like the planck rev6 that use a matrix that doesn't match the key layout, but reformats it to match, for inspiration.  Or the GMMK pro.

Reply:
Hey @drashna do I have to reformat all the keymap.c files also or only the header file?

Reply:
It would be all.  

The goal of the `LAYOUT` stuff is to display the keyboard matrix in a human friendly layout, one that matches what the keyboard looks like to a human.  

Changing that here means that you'd need to change it for all the keymaps, too.